store/internal/maps/maps.go

```diff
@@ -36,11 +36,11 @@ func (sm *merkleMap) set(key string, value []byte) {
 
 	// The value is hashed, so you can check for equality with a cached value (say)
 	// and make a determination to fetch or not.
-	vhash := sha256.Sum256(value)
```

Comment:
why migrate to cosmos crypto here?

Reply:
because the sha256 code was a copy&paste of what cometbft has. The idea is that also comet uses cosmos/crypto for the common crypto code.

Reply:
but why not use sha256 directly instead of a wrapper?